doc/requirements.txt

```diff
@@ -9,12 +9,12 @@ sphinxcontrib-bibtex
 myst-parser  # recommark is deprecated, https://stackoverflow.com/a/71660856/13573820
 sphinx-copybutton
 
-pypsa >=0.24, <0.25
+pypsa >=0.25, <=0.30
```

Comment:
Could it make sense to add a comment on the reason on this restriction? Backward compatibility is an issue for PyPSA and we get regular questions on the reasons why do we relay on it's particular version.

Reply:
Actually, I completely agree. Moreover, not sure for which reason we have pypsa as a dependency of the doc environment. But that is probably something worth a dedicated PR. For now, I think it's perfectly alright to leave it as-is.